recipe/meta.yaml

```diff
@@ -10,27 +10,30 @@ source:
   sha256: 8281600da180ec8ae35427a4ab4f7b82bfec1e3d1e52f80cb60ea82b9512501c
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
-    - regex
+    - python
+    - regex >=2022.3.15
 
 test:
   imports:
     - parsimonious
   requires:
+    - pip
     - pytest
   source_files:
     - parsimonious/tests
```

Comment:
```suggestion
```
This is not required because the files are already part of the wheel, see https://inspector.pypi.io/project/parsimonious/0.10.0/packages/aa/0f/c8b64d9b54ea631fcad4e9e3c8dbe8c11bb32a623be94f22974c88e71eaf/parsimonious-0.10.0-py3-none-any.whl/.

Reply:
that is interesting, thanks!